<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grayScaleClassClass</class>
 <widget class="QWidget" name="grayScaleClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>grayScaleClass</string>
  </property>
  <widget class="QLabel" name="label_showProcessImage">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>461</width>
     <height>461</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>第1点X坐标：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>第1点Y坐标：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>第2点X坐标：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>第2点Y坐标：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oneXlineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="oneYlineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="twoXlineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="twoYlineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enterpushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Adobe Arabic&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelpushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Adobe Arabic&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>灰度拉伸参数</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
